extensions/amp-animation/0.1/web-animations.js

```diff
@@ -761,18 +761,37 @@ export class MeasureScanner extends Scanner {
           'Both "selector" and "target" are not allowed');
       targets = this.css_.queryElements(spec.selector);
       if (targets.length == 0) {
-        user().warn(TAG, `Target not found: "${spec.selector}"`);
+        const doc = this.css_.rootNode_;
+        var allNodes = doc.getElementsByTagName('*');
```

Comment:
Ooof, this is all going to be really expensive.

Reply:
Yes, this is a naive approach. Potentially smarter approaches is holding a reference to all attached Shadow DOMS. Skips having to find them?